CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
+* Plug insert streams into backend ([#4512](https://github.com/confluentinc/ksql/pull/4512)) ([2482f0f](https://github.com/confluentinc/ksql/commit/2482f0f))
+* Provide upper limit on number of push queries ([#4581](https://github.com/confluentinc/ksql/pull/4581)) ([2cd66c7](https://github.com/confluentinc/ksql/commit/2cd66c7))
+* Simple perf tool for new API ([#4590](https://github.com/confluentinc/ksql/pull/4590)) ([ee7ca8a](https://github.com/confluentinc/ksql/commit/ee7ca8a))
+* support custom column widths in cli ([#4616](https://github.com/confluentinc/ksql/pull/4616)) ([42282cd](https://github.com/confluentinc/ksql/commit/42282cd))
+* support JSON schema inference in ksqlDB ([#4563](https://github.com/confluentinc/ksql/pull/4563)) ([fcd6c7f](https://github.com/confluentinc/ksql/commit/fcd6c7f))
+* support Protobuf in ksqlDB ([#4469](https://github.com/confluentinc/ksql/pull/4469)) ([a77cebe](https://github.com/confluentinc/ksql/commit/a77cebe))
+
+
+
+### Bug Fixes
+
+* build ([#4767](https://github.com/confluentinc/ksql/pull/4767)) ([6f45803](https://github.com/confluentinc/ksql/commit/6f45803))
+* fix test compile goal ([#4769](https://github.com/confluentinc/ksql/pull/4769)) ([4e8e601](https://github.com/confluentinc/ksql/commit/4e8e601))
+* fix the half-fixed json decimal tests ([#4768](https://github.com/confluentinc/ksql/pull/4768)) ([d87b97d](https://github.com/confluentinc/ksql/commit/d87b97d))
+* backport fixes from query close ([#4704](https://github.com/confluentinc/ksql/pull/4704)) ([1ac7f13](https://github.com/confluentinc/ksql/commit/1ac7f13)), closes [#4643](https://github.com/confluentinc/ksql/issues/4643) [#4658](https://github.com/confluentinc/ksql/issues/4658)
+* change default exception handling for timestamp extractors ([#4632](https://github.com/confluentinc/ksql/pull/4632)) ([1576af0](https://github.com/confluentinc/ksql/commit/1576af0))
+* create schemas at topic creation ([#4717](https://github.com/confluentinc/ksql/pull/4717)) ([514025d](https://github.com/confluentinc/ksql/commit/514025d))
+* decimals in structs should display as numeric ([#4165](https://github.com/confluentinc/ksql/pull/4165)) ([#4716](https://github.com/confluentinc/ksql/issues/4716)) ([7585d3c](https://github.com/confluentinc/ksql/commit/7585d3c))
+* don't display decimals in scientific notation in CLI ([#4723](https://github.com/confluentinc/ksql/pull/4723)) ([3626f42](https://github.com/confluentinc/ksql/commit/3626f42))
+* stop logging about command topic creation on startup if exists (MINOR) ([#4709](https://github.com/confluentinc/ksql/pull/4709)) ([f4cec0a](https://github.com/confluentinc/ksql/commit/f4cec0a))
+* add functional-test dependencies to Docker module ([#4586](https://github.com/confluentinc/ksql/pull/4586)) ([b023cd0](https://github.com/confluentinc/ksql/commit/b023cd0)), closes [#4584](https://github.com/confluentinc/ksql/issues/4584)
```

Comment:
is this an internal thing or related to our testing tool?

Reply:
It's a fix to the testing tool, and it's actually the reason the 0.7.1 point release was created. Which brings up a good question -- should changes included in the 0.7.1 release also be included in the 0.8.0 changelog, or no? (Asked differently, should the 0.8.0 changelog include new changes since 0.7.0 or only new changes since 0.7.1?)

Reply:
Standard practice is to only include new things since the last point release. So in this case only new stuff after 0.7.1.

Reply:
Makes sense. Removed the 0.7.1 changes from the 0.8.0 changelog.